plugins/labormanager.cpp

```diff
@@ -2156,10 +2300,16 @@ class AutoLaborManager {
             {
                 skill_level = Units::getEffectiveSkill(d->dwarf, skill);
                 xp = Units::getExperience(d->dwarf, skill, false);
+
+                for (int pa = 0; pa < 6; pa++) 
```

Comment:
Here's the output on my end:
```
Contains trailing whitespace: lines 1829, 2304, 2307, 2582-2583, 2710-2711, 2763-2766, 2896
```
It doesn't look like you touched 1829 here, though, and it's fine on dfhack/dfhack:develop. It could be due to rebasing, although I'm not really sure.